values.yaml

```diff
@@ -41,6 +41,24 @@ cmak:
     saslMechanism: null
     jaasConfig: null
 
+  # Cosumer Properties
+  consumerProperties: {}
```

Comment:
`cmak` section is a section containin only those settings that are configured via UI and are stored to zookeeper.

Instead, `consumer properties` are settings to the cmak application itself and should be put under `ui` key.

Reply:
we can move the `consumerProperties` into the ui key with no issues. but putting the ssl config under the `consumerPorperties` will make the rage function in the helper have to exclude it from the loop. that's why i put it under a separate key...do you think moving both these to be nested under ui instead of cmak is enough? or we should we make ssl be under consumerProperties and make the helper more complex....

Reply:
BTW in the values file th ui section has this comment above it: ```# K8S related settings for CMAK UI container``` so i think that' why i did not put these settings there as they are not related to k8s they are related to cmak itself...

Reply:
might be more reasonable to change the comment above the cmak section the state something like ```# CMAK application configuration settings```

Reply:
@eshepelyuk LMK what you think so i can push this forward...

Reply:
@eshepelyuk can we complete the discussion here? i would prefer using a proper release as at the moment i'm using my local branch on y clusters...